### ISSUE TITLE
the "acc" output in training a Keras model

### ISSUE THREAD

**Issue by surfreta** [fchollet/keras#4896]
Hi

During train a Keras model, the training output looks like as follows
`4375/4375 [==============================] - 821s - loss: 0.1497 - acc: 0.9749`
I know loss is for the cost, but what does “acc” mean here. In the training setup, I did not include the validation split. Therefore, I am not very clear the meaning of “acc” here. Thanks. 

surfreta

**Comment by ozancaglayan** [comment]
That is the training accuracy.

**Comment by saddy001** [comment]
- And I assume it's 0-1-scaled, so acc * 100 is percent?
